modules/core/src/main/java/org/apache/ignite/spi/systemview/view/CacheView.java

```diff
@@ -331,7 +336,46 @@ public String cacheWriterFactory() {
 
     /** @see CacheConfiguration#getExpiryPolicyFactory() */
     public String expiryPolicyFactory() {
-        return toStringSafe(cache.cacheConfiguration().getExpiryPolicyFactory());
+        if (cache.cacheConfiguration().getExpiryPolicyFactory() == null)
+            return null;
+
+        ExpiryPolicy expiryPlc = (ExpiryPolicy)cache.cacheConfiguration().getExpiryPolicyFactory().create();
```

Comment:
We should also expose information about factory (according to method name)

Reply:
Added information about the factory as well